x-pack/plugins/security_solution/public/resolver/view/panel.test.tsx

```diff
@@ -111,16 +116,40 @@ describe(`Resolver: when analyzing a tree with no ancestors and two children and
         wordBreaks: 2,
       });
     });
-    it('should allow all node details to be copied', async () => {
-      const copyableFields = await simulator().resolve('resolver:panel:copyable-field');
+    describe.each([...originEventDetailEntries])(
```

Comment:
These tests now use a statically defined map of fields and expected values. The test finds the `dt` for each field and then finds the related `dd`s. From there it finds a special 'hover area' (via `data-test-subj`) and simulates a `mouseenter` on it. This is because the feature work by adding event listeners to `div`s. There is no way for the user to know that the `div`s are interactable.

Finally the test clicks a button and checks that the clipboard was written to.